examples/constant-force-optical-trap/output/counts-expected-0.out

```diff
@@ -1,50 +1,50 @@
- 494.896      3701.003313        58.541052
```

Comment:
What's the point of including "expected output" targets if this changes even if we are only changing the efficiency and robustness, rather than the expected result? Shouldn't we just adjust the comparison tolerance and keep a single, well-converged set of results here?

Reply:
Reproducibility, basically.  They should get the same answer as in the code.  

I don't know how to have git compare "up to a number".